runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -3683,7 +3683,7 @@ J9::Z::CodeGenerator::suppressInliningOfRecognizedMethod(TR::RecognizedMethod me
    if (self()->isMethodInAtomicLongGroup(method))
       return true;
 
-   if (!self()->comp()->compileRelocatableCode() && !self()->comp()->getOption(TR_DisableDFP) && TR::Compiler->target.cpu.getSupportsDecimalFloatingPointFacility())
+   if (!(self()->comp()->compileRelocatableCode() || self()->comp()->isOutOfProcessCompilation()) && !self()->comp()->getOption(TR_DisableDFP) && TR::Compiler->target.cpu.getSupportsDecimalFloatingPointFacility())
```

Comment:
The change here and on line 4121 was added to avoid a test crash. However this may cause a performance gap. I'll perform a few acmeAir runs on OpenJ9 to evaluate if there's a performance gap created when DFP instructions are disabled.

Reply:
I'm no longer able to see a crash when backing out this code. In terms of performance, the difference is negligible when this code is executed vs not executed in just OpenJ9. So I will go ahead and remove the checks for DFPs.

Reply:
Note, I ran `sanity.functional.regular` tests on a z13 machine, and `sanity.functional.native` tests on a z14 machine to verify no problem occurs.